tests/rptest/redpanda_cloud_tests/simple_redpanda_service_cloud_test.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import random
+from typing import Any
+from ducktape.tests.test import TestContext
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.services.redpanda import RedpandaServiceCloud
+from rptest.services.redpanda_cloud import get_config_profile_name
+
+
+class SimpleRedpandaServiceCloudTest(RedpandaTest):
```

Comment:
This should be a RedpandaCloudTest, then `self.redpanda` is guaranteed to be a RedpandaServiceCloud.

Reply:
i've also renamed this class to `SelfRedpandaCloudTest`